ansible/create_template_box.yml

```diff
@@ -6,29 +6,25 @@
     - name: fedora_version
       prompt: fedora
       private: false
-      default: 27
+      default: 29
     - name: git_branch
       prompt: "git branch"
       private: false
       default: "master"
   vars_files:
     - vars/fedora/{{ fedora_version }}.yml
     - vars/ipa_branches/{{ git_branch }}.yml
+  gather_facts: false
   pre_tasks:
+    - name: install Python 3 deps for ansible modules
+      raw: dnf install -y python3-dnf python3-libselinux
```

Comment:
This is intended to be executed on developer's machine, so this must add `become: yes` parameter.

Also , it could use `dnf` instead of `raw`.